# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../sysvinit/sysvinit-2.88dsf-distro-opensde.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- sysvinit-2.88dsf/src/Makefile.orig	2010-04-11 11:30:27.000000000 +0200
+++ sysvinit-2.88dsf/src/Makefile	2011-09-13 13:15:23.616084369 +0200
@@ -53,6 +53,14 @@
 MAN8	+= sulogin.8
 endif
 
+ifeq ($(DISTRO),OpenSDE)
+BIN	+= mountpoint
+SBIN	+= sulogin bootlogd
+USRBIN	+= utmpdump
+MAN1	+= utmpdump.1 mountpoint.1
+MAN8	+= sulogin.8 bootlogd.8
+endif
+
 ID		= $(shell id -u)
 BIN_OWNER	= root
 BIN_GROUP	= root
